frontend/src/components/CircularMenu/CircularMenu.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import "./CircularMenu.scss";
+import PropTypes from "prop-types";
+
+const CircularMenu = ({
+  itemsAmount,
+  x,
+  y,
+  row,
+  column,
+  updateBoard,
+  hideMenu
+}) => {
+  const tilesArray = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const [opened, setOpened] = useState(false);
```

Comment:
Generally, a good naming practice is to name booleans with some form of be + adjective/adverb. So in this case `isOpen` and `setIsOpen`. But this is highly subjective so whatever looks good to you.